assets/config/helpers.scss

```diff
@@ -66,3 +66,10 @@
     background: $value !important;
   }
 }
+
+.offscreen {
```

Comment:
For what are you using it for ?

Reply:
basolutely nothig, it was a test for a nice mobile animation for different heights. removed